ext/pdo/pdo_stmt.c

```diff
@@ -872,11 +852,13 @@ static bool do_fetch(pdo_stmt_t *stmt, zval *return_value, enum pdo_fetch_type h
 
 		case PDO_FETCH_FUNC:
 			/* TODO: Make this an assertion and ensure this is true higher up? */
-			if (!ZEND_FCI_INITIALIZED(stmt->fetch.func.fci)) {
+			if (!ZEND_FCC_INITIALIZED(stmt->fetch.func.fcc)) {
 				/* TODO ArgumentCountError? */
 				pdo_raise_impl_error(stmt->dbh, stmt, "HY000", "No fetch function specified");
 				return false;
 			}
+			/* We can probably infer items than stmt->column_count for some cases */
```

Comment:
I didn't understand this sentence. Something gramatically makes this difficult for me to understand, I guess the "than" is out of place?